morevna-ep4.msrt

```diff
@@ -271,7 +271,7 @@
 55
 00:05:47,300 --> 00:05:49,640
 [rus] И Марью Моревну я тоже спасу!
-[eng] And I'll save Marya Morevna as well!
+[eng] And I'll save Marya Morevna!
```

Comment:
To me, saying "as well" makes it sound as if he is saving Marya Morevna in addition to some other people. I think this works quite well without anything extra.